modules/nf-core/diamond/blastp/tests/main.nf.test

```diff
@@ -0,0 +1,102 @@
+nextflow_process {
+
+    name "Test Process DIAMOND_BLASTP"
+    script "../main.nf"
+    process "DIAMOND_BLASTP"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "diamond"
+    tag "diamond/blastp"
+
+    setup {
+        run("DIAMOND_MAKEDB") {
+            script "../../makedb/main.nf"
+            process {
+                """
+                input[0] = [ [id:'test'], [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ] ]
+                """
+            }
+        }
+    }
+
+    test("Should search for protein hits against a DIAMOND db and return a tab separated output file of hits") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [ [id:'test'], file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
+                input[1] = DIAMOND_MAKEDB.out.db
+                input[2] = 'txt'
+                input[3] = 'qseqid qlen'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.txt },
```

Comment:
Is it necessary to check if the file is there if you also check for content?

Reply:
True. Not needed